client/my-sites/theme/style.scss

```diff
@@ -20,6 +20,28 @@ body.is-section-theme-i4 {
 		}
 	}
 
+	.theme__sheet-action-bar-container {
+		box-shadow: 0 0 0 1px var(--color-border-subtle);
+	}
+
+	.theme__sheet-action-bar {
+		box-shadow: none;
+		box-sizing: content-box;
+		margin: 0 auto;
+		max-width: 1462px;
```

Comment:
Is it possible to define the value for this one so we can have a single source of it?

Reply:
Good point!